examples/terraform/vmware-cloud-director/output.tf

```diff
@@ -23,6 +23,10 @@ output "kubeone_api" {
   }
 }
 
+output "ssh_commands" {
+  value = var.enable_bastion_host ? formatlist("ssh -J ${var.ssh_username}@${local.external_network_ip}:${var.bastion_host_ssh_port} ${var.ssh_username}@%s", vcd_vapp_vm.control_plane.*.network.0.ip) : formatlist("ssh ubuntu@%s", vcd_vapp_vm.control_plane.*.network.0.ip)
```

Comment:
Instead of ubuntu it should be var.ssh_username

Reply:
Good point, fixed 👍